### PR TITLE
Improved readme and changelog

### PR DESCRIPTION
**Pull request by ricardosantosalves**

### CODE REVIEW

File: .gitchangelog.rc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     ]),
+    ('Other', None ## Match all lines
+     ),
+
+]
+
+
+ignore_regexps = [
+    r'^Merge*',
+    r'^Automatic merge*',
```
Comment (kammala) [inline .gitchangelog.rc]: is it regular expression or wildcard?
for regexp it should be `.*` instead of just `*`

File: Makefile [hunk 2]
```diff
@@ -60,3 +60,6 @@ docker/%:
 	$(DOCKER_COMPOSE) run --rm app make $*
 
 build: clean venv tox
+
+changelog:
+	gitchangelog
```
Comment (kammala) [inline Makefile]: did you mean `venv/bin/changelog`?

and if so, should we add `venv` as dependency for `changelog` target?

Comment (ricardosantosalves) [reply Makefile]: The command is `venv/bin/gitchangelog`. I'll adjust it. Also the dependency.

File: .gitchangelog.rc (hunk 1 above)
Comment (ricardosantosalves) [reply .gitchangelog.rc]: Well, changed a bit to actually be more strict. But tested and working properly.

File: Makefile (hunk 2 above)
Comment (ricardosantosalves) [reply Makefile]: Fixed.

**Review by kammala — APPROVED** [review]

**Review by kammala — APPROVED** [review]

**Pull request merged**